shubhali/curdop/views/index.ejs

```diff
@@ -11,72 +11,62 @@
-                    <form action="/" id="EmployeeForm" class="form-horizontal" method="post"  role="form">
-                      <div id="signupalert" style="display:none" class="alert alert-danger">
-                        <p>Error:</p>
-                        <span></span>
-                      </div>
-                      <div class="form-group">
-                        <label for="firstname" class="col-md-3 control-label">Name</label>
-                        <div class="col-md-9">
-                          <input type="text" class="form-control" name="uname" placeholder="Enter Your Name">
-                        </div>
-                      </div>
-                      <div class="form-group">
-                        <label for="lastname" class="col-md-3 control-label">Last Name</label>
-                        <div class="col-md-9">
-                          <input type="text" class="form-control" name="lastname" placeholder="Lastname">
-                        </div>
-                      </div>
-                      <div class="form-group">
-                        <label for="email" class="col-md-3 control-label">Email</label>
-                        <div class="col-md-9">
-                          <input type="text" class="form-control" name="email" placeholder="Email Address">
-                        </div>
-                      </div>
-                      <div class="form-group">
-                          <label for="password" class="col-md-3 control-label">Phone Number</label>
-                          <div class="col-md-9">
-                            <input type="text" class="form-control" name="phone" placeholder="Enter phone number">
-                          </div>
-                      </div>
-                      <div class="form-group">
-                        <label for="icode" class="col-md-3 control-label"> Branch</label>
-                        <div class="col-md-9">
-                          <input type="text" class="form-control" name="field" placeholder="Enter your field">
-                        </div>
-                      </div>
-                      <div class="form-group">
-                        <label for="city" class="col-md-3 control-label"> City</label>
-                        <div class="col-md-9">
-                          <input type="text" class="form-control" name="city" placeholder="Enter your city name">
-                        </div>
-                      </div>
-                      <div class="form-group">
-                        <div class="col-md-offset-3 col-md-9">
-                          <input type="submit" name="submit" value="submit" class="btn btn-primary">   
-                        </div>
-                     </div>
-                 </form>
+        <div class="open-button" onclick="openForm()">Add employee</div>
+        <div id="signupbox" style="margin-top:50px" class="mainbox form-popup col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2">
```

Comment:
after colon add space make it as style="margin-top: 50px" in all places where u have used inline style